packages/js/src/ui/context/AppearanceContext.tsx

```diff
@@ -116,7 +119,7 @@ export const AppearanceProvider = (props: AppearanceProviderProps) => {
   return (
     <AppearanceContext.Provider
       value={{
-        elements,
+        elements: allElements,
```

Comment:
previously, it only included the elements from the props and didn't include the elements from the baseTheme appearance.